magellan-lint/src/main/java/com/wealthfront/magellan/AvoidUsingActivity.kt

```diff
@@ -34,13 +34,13 @@ internal class AvoidUsingActivity : Detector(), Detector.UastScanner {
 
     override fun visitCallExpression(node: UCallExpression) {
       if (node.isSubtypeOfNavigableCompat() && node.receiverType?.canonicalText == ACTIVITY)
```

Comment:
nit: we probably want braces around this if statement

Reply:
d